### ISSUE TITLE
Example showing subscriptions

### ISSUE THREAD

**Issue by florapdx** [4Catalyzer/found-relay#99]
Hi @taion! I want to first thank you for all of your work building routing solutions for Relay. Much appreciated! 

I know that documentation is forthcoming for this project, but I'm wondering if it would be relatively trivial for you to add a subscription example to one of the existing Relay Modern examples? It's not clear to me, for example, how found-relay's route config should coexist with Relay's `requestSubscription`.

**Comment by taion** [comment]
Thanks for your kind words.

Subscription support is orthogonal to the routing concerns here. If you've used Relay Subscriptions with Relay Classic, the semantics are the same. You call `requestSubscription` in `componentDidMount` of the component that needs to subscribe, dispose the subscription in `componentWillUnmount`, and optionally update the subscription (dispose and re-create) if the component props have changed.

In other words it looks the same regardless of your routing approach or lack thereof.
